src/main/java/ml/shifu/shifu/core/dtrain/dt/DTMaster.java

```diff
@@ -234,6 +234,10 @@
      */
     private Queue<TreeNode> toDoQueue;
 
+    private DTEarlyStopDecider dtEarlyStopDecider;
```

Comment:
Please add comment for these two fields, especially these two fields only works in GBDT